nixos/modules/services/networking/keepalived/default.nix

```diff
@@ -110,6 +152,14 @@ in
         '';
       };
 
+      enableScriptSecurity = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Don't run scripts configured to be run as root if any part of the path.
```

Comment:
This should be "Don't run scripts configured to be run as root if any part of the path is writable by a non-root user."